tests/reproducible_builds/check_reproducible_build.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/sh
+
+if [ -d "/home/$USER/zephyr_src1" ]; then
+    rm -rf /home/$USER/zephyr_src1
+fi
+
+if [ -d "/home/$USER/zephyr_src2" ]; then
+    rm -rf /home/$USER/zephyr_src2
+fi
+
+mkdir /home/$USER/zephyr_src1
+echo "Cloning latest code in zephyr_src1"
+cd /home/$USER/zephyr_src1
+git clone https://github.com/zephyrproject-rtos/zephyr.git
+
+
+
+mkdir /home/$USER/zephyr_src2
+echo "Cloning latest code in zephyr_source2"
```

Comment:
why are we cloning Zephyr twice here? It is enough to build an application outside of Zephyr tree in 2 different location to test this.